roles/setup/tasks/setup-Debian.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Install receptor packages
```

Comment:
I think we probably want to drop this task as we don't package receptor for Debian systems. So for Debian the expectation is that users will use local_receptor option